tests/handle_error_3/lib.sh

```diff
@@ -45,31 +45,31 @@ function clean_table() {
 }
 
 function get_start_location() {
-	location=$($PWD/bin/dmctl.test DEVEL -master-addr=$1 \
+	location=$($PWD/bin/dmctl.test DEVEL --master-addr=$1 \
 		query-status test -s $2 |
 		grep Location |
 		gawk 'match($0,/startLocation: \[position: \((.*), (.*)\), gtid-set: (.*)\], endLocation: \[position: \((.*), (.*)\), gtid-set: (.*)\]/,a) {printf "%s:%s", a[1], a[2]}')
 	echo ${location}
 }
 
 function get_end_location() {
-	location=$($PWD/bin/dmctl.test DEVEL -master-addr=$1 \
+	location=$($PWD/bin/dmctl.test DEVEL --master-addr=$1 \
 		query-status test -s $2 |
 		grep Location |
 		gawk 'match($0,/startLocation: \[position: \((.*), (.*)\), gtid-set: (.*)\], endLocation: \[position: \((.*), (.*)\), gtid-set: (.*)\]/,a) {printf "%s:%s", a[4], a[5]}')
 	echo ${location}
 }
 
 function get_start_pos() {
-	pos=$($PWD/bin/dmctl.test DEVEL -master-addr=$1 \
+	pos=$($PWD/bin/dmctl.test DEVEL --master-addr=$1 \
 		query-status test -s $2 |
 		grep Location |
 		gawk 'match($0,/startLocation: \[position: \((.*), (.*)\), gtid-set: (.*)\], endLocation: \[position: \((.*), (.*)\), gtid-set: (.*)\]/,a) {print a[2]}')
 	echo ${pos}
 }
 
 function get_start_name() {
-	pos=$($PWD/bin/dmctl.test DEVEL -master-addr=$1 \
```

Comment:
I see our stuff using one `-` sometimes, so we might keep compatibility

Reply:
addressed in https://github.com/pingcap/dm/pull/1750/commits/09244541025887ec0ec3e8777a1f67c5cec5b411